src/test/shared/clients/defaultEcsClient.test.ts

```diff
@@ -1,247 +0,0 @@
-/*!
```

Comment:
Nice. These tests were redundant.
We do need to create integration tests (eventually) to test our client wrappers with real clients. But for now I think if the wrappers are very simple (which they are in this case) then using a mocking framework is appropriate.